Automattic/woocommerce-payments
# Record a Tracks event when viewing the order success page

#### Changes proposed in this Pull Request
Record a Tracks event when viewing the Thank you page after a successful order









#### Testing instructions
1. Place an order using WooPayments standard checkout flow
2. In ~5 minutes check MC's Tracks Live view for the presence of `wcpay_order_success_page_view`.
3. Place an order using WooPay and check MC Tracks Live for the same event.
4. Place an order using a non WooPayments gateway (eg: Stripe, Cash on Delivery, Check Payments).
5. Make sure the above event does not appear on MC. 






-------------------


**Post merge**
